src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -48,7 +48,8 @@
  * @checkstyle ClassFanOutComplexity (10 lines)
  * @todo #923 Let's implement RtRepo.languages().
  *  Now it returns empty iterable but should return list of repo languages.
- *  Uncomment test RtRepoTest.iteratesLanguages() when done.
```

Comment:
Just add the new puzzle with the current ticket number (`@todo #955`), and indicate the extra work that need to be done. Leave the existing puzzle as it is.